pkg/util/util.go

```diff
@@ -44,8 +46,15 @@ func ConfigureLogging(logLevel string) error {
 
 // Hasher returns a hash function, used in snapshotting to determine if a file has changed
 func Hasher() func(string) (string, error) {
+	pool := sync.Pool{
+		New: func() interface{} {
+			b := make([]byte, highwayhash.Size*10*1024)
+			return &b
+		},
+	}
+	key := make([]byte, highwayhash.Size)
 	hasher := func(p string) (string, error) {
-		h := md5.New()
+		h, _ := highwayhash.New(key)
```

Comment:
Check error. Can you explain or add link to benefits os `highwayhash` instead of `md5`.
Any disadvantages? Is this project being actively worked on?